### ISSUE TITLE
Add the F9 shortcut for toggling the sidebar

### ISSUE THREAD

**Issue by nekohayo** [phase1geo/Minder#252]
Please add this standard/widespread keyboard shortcut :)
Additionally, the "Properties" button should probably have a more descriptive, action-oriented tooltip, like "Show the properties sidebar" and "Hide the properties sidebar", and use a ToggleButton instead of a regular button with icons that change.

**Comment by phase1geo** [comment]
The icon should already change when clicked.

I can add the other enhancements.

**Comment by nekohayo** [comment]
Yeah, but what I'm saying is that changing the icon instead of using a togglebutton is counter to common design patterns in GTK/GNOME/elementary apps. If it is not simply a checkbox inside a hamburger menu, I would expect that button to be a togglebutton with a symbolic sidebar icon...

**Comment by phase1geo** [comment]
There isn't a prescribed design pattern for sidebar show/hide in elementary apps that I am aware of. Additionally, all icons in the header bar should not be symbolic icons to conform to the elementary HIG. That is a Gnome convention.

**Comment by phase1geo** [comment]
I have changed the keyboard shortcut from Control-| to F9 to show/hide the properties sidebar. I have changed the properties button to a toggle (though I am still changing the icon direction based on state). I have also changed this icon to a symbolic icon for non-elementary themed systems (along with the focus and search icons in the header bar)

All of these changes have been committed to the devel branch and will be generally available in the next stable release.
